infra/make-index.rkt

```diff
@@ -256,9 +256,9 @@
        (script
         "window.addEventListener('load', function(){draw_results(d3.select('#accuracy-graph'), d3.select('#speed-graph'))})"))
       (table ((id "reports"))
-             ,@(apply append
-                      (for/list ([rows (append mainline-infos other-infos)])
-                        (print-rows rows #:name (dict-ref (first rows) 'branch)))))))
+             ,@(for*/list ([rows (append mainline-infos other-infos)]
+                           [v (in-list (print-rows rows #:name (dict-ref (first rows) 'branch)))])
+                 v))))
```

Comment:
@jackfirth Not a big fan of this rewrite rule. The result might be shorter but it's definitely more "clever" code, in a bad sense.

Reply:
I could get behind that. It's less than optimal that it moves the entire body into a clause.

Would you mind opening an issue in the Resyntax repository for this?

Reply:
Done: https://github.com/jackfirth/resyntax/issues/435